build/components/render/web-design-system.html

```diff
@@ -0,0 +1,15 @@
+
```

Comment:
This is the layout template I am using for the Standards specific components.

Also, see https://github.com/18F/web-design-standards/issues/1518. We have not come to a final decision on this yet and I should probably change the name to `web-design-standards` to cause less confusion.